sample/src/main/java/com/github/pedrovgs/sample/activity/PlacesSampleActivity.java

```diff
@@ -69,6 +77,22 @@
     initializeFragments();
     initializeListView();
     initializeDraggablePanel();
+    configNavigationDrawer();
+  }
+
+  /**
+   * Sync the drawerToggle
+   */
+  @Override
+  protected void onPostCreate(Bundle savedInstanceState) {
+    super.onPostCreate(savedInstanceState);
+    mDrawerToggle.syncState();
```

Comment:
This repository doesn't use "m" notation. Please review your variables and rename them without the "m" prefix.